include/zephyr/bluetooth/audio/cap.h

```diff
@@ -636,6 +636,244 @@ struct bt_cap_broadcast_to_unicast_param {
+
+	/** The number of members in @p members */
+	size_t count;
+};
+
+/**
+ * @brief Stops the reception of broadcast audio on one or more remote Common Audio Profile
+ * Acceptors
+ *
+ * @param param The parameters to stop the broadcast audio
+ *
+ * @return 0 on success or negative error value on failure.
+ */
+int bt_cap_commander_broadcast_reception_stop(
+	const struct bt_cap_commander_broadcast_reception_stop_param *param);
+
+/** Parameters for changing absolute volume  */
+struct bt_cap_commander_change_volume_param {
+	/** The type of the set. */
+	enum bt_cap_set_type type;
+
+	/** Coordinated or ad-hoc set member. */
+	union bt_cap_set_member *members;
+
+	/** The number of members in @p members */
+	size_t count;
+
+	/** The absolute volume to set */
+	uint8_t volume;
+};
+
+/**
+ * @brief Change the volume on one or more Common Audio Profile Acceptors
+ *
+ * @param param The parameters for the volume change
+ *
+ * @return 0 on success or negative error value on failure.
+ */
+int bt_cap_commander_change_volume(const struct bt_cap_commander_change_volume_param *param);
+
+struct bt_cap_commander_change_volume_offset_member_param {
+	/** Coordinated or ad-hoc set member. */
+	union bt_cap_set_member member;
+
+	/**
+	 * @brief  The offset to set
+	 *
+	 * Value shall be between @ref BT_VOCS_MIN_OFFSET and @ref BT_VOCS_MAX_OFFSET
+	 */
+	int16_t offset;
+};
+
+/** Parameters for changing volume offset */
+struct bt_cap_commander_change_volume_offset_param {
+	/** The type of the set. */
+	enum bt_cap_set_type type;
+
+	/** The set of devices for this procedure */
+	struct bt_cap_commander_change_volume_offset_member_param *param;
+
+	/** The number of parameters in @p param */
+	size_t count;
+};
+
+/**
+ * @brief Change the volume offset on one or more Common Audio Profile Acceptors
+ *
+ * @param param The parameters for the volume offset change
+ *
+ * @return 0 on success or negative error value on failure.
+ */
+int bt_cap_commander_change_volume_offset(
+	const struct bt_cap_commander_change_volume_offset_param *param);
+
+/** Parameters for changing volume mute state */
+struct bt_cap_commander_change_volume_mute_state_param {
+	/** The type of the set. */
+	enum bt_cap_set_type type;
+
+	/** Coordinated or ad-hoc set member. */
+	union bt_cap_set_member *members;
+
+	/** The number of members in @p members */
+	size_t count;
+
+	/**
+	 * @brief The volume mute state to set
+	 *
+	 * true to mute, and false to unmute
+	 */
+	bool mute;
+};
+
+/**
+ * @brief Change the volume mute state on one or more Common Audio Profile Acceptors
+ *
+ * @param param The parameters for the volume mute state change
+ *
+ * @return 0 on success or negative error value on failure.
+ */
+int bt_cap_commander_change_volume_mute_state(
+	const struct bt_cap_commander_change_volume_mute_state_param *param);
+
+/** Parameters for changing microphone mute state */
+struct bt_cap_commander_change_microphone_mute_state_param {
```

Comment:
Noted that this is identical to the bt_cap_commander_change_volume_mute_state_param. I think it is good to keep the two definitions since they are used in different procedures

Reply:
Indeed